### PR TITLE
Make the failover scripts usable "out of the box".

### PR DESCRIPTION
**Pull request by gregcorbett**
To reduce the steps between downloading the scripts and importing a DB dump.

### CODE REVIEW

**Review by ineilson — COMMENTED** [review]
Is it necessary to give 'other' execute?

**Comment by gregcorbett** [comment]
> Is it necessary to give 'other' execute?

So it seems that is a git artifact https://medium.com/@tahteche/how-git-treats-changes-in-file-permissions-f71874ca239d

**Comment by ineilson** [comment]
> So it seems that is a git artifact h

Seems curious, but I'm sure there's a good reason 🤔

File: importDBdmpFile/dropGocdbUser2.sh [hunk 1]
```diff
@@ -19,9 +19,8 @@
 #
 # RESULT=$(sqlplus system/XXXXXXX <<- ENDSQL
 
-# PASSFILE is in "parfile" format for impdp
-PASSFILE=/root/gocdb-failover-scripts/importDBdmpFile/pass_file
-USER_PASS=$(cat "${PASSFILE}" | grep "^userid=" | cut -d "=" -f 2)
+# pass_file is in "parfile" format for impdp
+USER_PASS=$(cat "pass_file" | grep "^userid=" | cut -d "=" -f 2)
 
 # Need to exit gracefully if pass_file doesn't exist or
```
Comment (ineilson) [inline importDBdmpFile/dropGocdbUser2.sh:25]: Is this implemented? Could add check to fail also if permissions on PASSFILE are too open.

Comment (gregcorbett) [reply importDBdmpFile/dropGocdbUser2.sh:25]: One can only hope :p. Feel like opening an issue about adding a check on pass_file's permissions?

**Review by ineilson — DISMISSED** [review]
Approved with the proviso I don't understand what github is doing with file permissions (which are obviously important in this case).

**Review by jrha — DISMISSED** [review]

File: importDBdmpFile/dropGocdbUser2.sh (hunk 1 above)
Comment (gregcorbett) [reply importDBdmpFile/dropGocdbUser2.sh:25]: Thanks for opening #5 😄

**Comment by gregcorbett** [comment]
rebase'd to resolve the README.md conflict.

**Pull request merged**